akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/directives/PathDirectivesExamplesSpec.scala

```diff
@@ -16,7 +16,7 @@ class PathDirectivesExamplesSpec extends RoutingSpec {
 
   //# path-dsl
   // matches /foo/
-  path("foo" /)
+  path("foo"./)
```

Comment:
Yes, I certainly would: this matches our usage of the duration DSL as well, where we have added the dot in (hopefully) all cases. Postfix operators are frowned upon for good reasons and importing the needed implicit just in the right scopes would be too bulky. What’s wrong with the line as it is now?